bgpd/bgp_evpn_mh.c

```diff
@@ -3589,12 +3601,24 @@ static struct bgp_evpn_es_evi *bgp_evpn_es_evi_new(struct bgp_evpn_es *es,
 	struct bgp_evpn_es_evi *es_evi;
 
 	es_evi = XCALLOC(MTYPE_BGP_EVPN_ES_EVI, sizeof(*es_evi));
+	if (!es_evi) {
```

Comment:
Usually, we don't add such checks for malloc()/calloc() foobar :shrug:

Reply:
xcalloc cannot fail, right?